diagrams/README.md

```diff
@@ -0,0 +1,12 @@
+### Introduction
```

Comment:
Can we put it in `docs/uml/` and add it as excluded? https://github.com/linkedin/venice/blob/main/docs/_config.yml#L8

Reply:
I moved them to `/assets/uml/`